const.go

```diff
@@ -48,3 +52,10 @@ var NotaryDefaultExpiries = map[string]time.Duration{
 	"snapshot":  NotarySnapshotExpiry,
 	"timestamp": NotaryTimestampExpiry,
 }
+
+// NotaryDefaultHashes is the construct used to configure the default
+// supported hash algorithms.
+var NotaryDefaultHashes = map[string]string{
```

Comment:
I agree with @cyli -- this will be clearer to understand and it'd also be nice to get rid of as many `"sha256"` and `"sha512"` strings in the code as possible.